BUILDSCRIPT_MULTIPROC.bash44

```diff
@@ -216,12 +209,13 @@ OBJ=("cryptbase/o/cryptbase.o"
      "headerframe/o/statics.o"
      "stickerframe/o/statics.o")
 
+num_jobs=${#SRC[@]}
```

Comment:
`num_jobs` is both branches of the if statement below, so it should be defined above. At the moment, the build fails when using Bash with a version below 4.4 because of this.